Assignment1/Assignment1_QNo3/SolAssignmentNo1_QNo3.rb

```diff
@@ -0,0 +1,23 @@
+# => 3. Find out how to merge two strings 'Hello', 'World'
+# => along with a space in between them?
+# => author : Hrisikesh Singha Chowdhury
+
+# => definition of custom class
+class CustomClass
+  # => Fn to implement buisness logic
+  def merge_string
+    puts "Enter first string"
+    fname = gets.chomp
+    puts "Enter second string"
+    lname = gets.chomp
+    # => Merging names with a space in between
+    puts "The required merged name is with white space in between #{fname} #{lname}"
```

Comment:
@HrisikeshSinghaChowdhury There is an inbuilt method 'join' in ruby, Please use that.

Reply:
the inputs are string not array